smooth/components/component_battery.py

```diff
@@ -0,0 +1,132 @@
+import sys
+
+import oemof.solph as solph
+from .component import Component
+from oemof.outputlib import views
+
+
+class Battery(Component):
+    """ Stationary battery is created through this class """
+
+    def __init__(self, params):
+
+        # Call the init function of the mother class.
+        Component.__init__(self)
+
+        """ PARAMETERS """
+        self.name = "Battery_default_name"
+
+        # Define the electric bus the battery is connected to.
+        self.bus_in_and_out = None
+        # Battery capacity (assuming all the capacity can be used) [Wh].
+        self.battery_capacity = 5000
+        # Initial State of charge [%].
+        self.soc_init = 0.5
```

Comment:
i think the comment is not correct. It would be in [-] otherwise you need to update the function where you create the oemof model